<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 30, 30);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="font">
           <attribute name="title">
            <string>字体</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Courier New&quot;;</string>
                  </property>
                  <property name="text">
                   <string>字型：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFontComboBox" name="cbFontType">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(208, 208, 208);
color: rgb(0, 0, 0);
font: 16pt &quot;Courier New&quot;;</string>
                  </property>
                  <property name="currentIndex">
                   <number>82</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Courier New&quot;;</string>
                  </property>
                  <property name="text">
                   <string>    字号：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbFontSize">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);
font: 75 16pt &quot;Courier New&quot;;
background-color: rgb(208, 208, 208);</string>
                  </property>
                  <property name="currentIndex">
                   <number>2</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>14</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>16</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>18</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>22</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>24</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>26</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>28</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>36</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>48</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>72</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>268</width>
                  <height>28</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelPreview">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;字体样式预览    font preview&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="codeType">
           <attribute name="title">
            <string>编码格式</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Courier New&quot;;</string>
                </property>
                <property name="text">
                 <string>以</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbCodeType">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
font: 75 16pt &quot;Courier New&quot;;
background-color: rgb(208, 208, 208);</string>
                </property>
                <item>
                 <property name="text">
                  <string>UTF-8</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>GBK</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>GB2312</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>UNICODE</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ANSI</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Courier New&quot;;</string>
                </property>
                <property name="text">
                 <string>的编码格式读取文件</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="ADBConfig">
           <attribute name="title">
            <string>ADB配置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Courier New&quot;;</string>
                </property>
                <property name="text">
                 <string>ADB工具路径：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="etADBPath">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(54, 54, 54);
color: rgb(255, 255, 255);
font: 16pt &quot;Courier New&quot;;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnSelectPath">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;Courier New&quot;;
background-color: rgb(212, 212, 212);</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>268</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnOK">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Courier New&quot;;
background-color: rgb(212, 212, 212);</string>
            </property>
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReset">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Courier New&quot;;
background-color: rgb(212, 212, 212);</string>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancel">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;Courier New&quot;;
background-color: rgb(212, 212, 212);</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
